Examples/Python/python/acts/examples/reconstruction.py

```diff
@@ -111,6 +111,12 @@
     defaults=[(None, None)],
 )
 
+TrackSelectorConfig = namedtuple(
+    "TrackSelectorConfig",
+    ["absEtaMax", "ptMin"],
+    defaults=[2.5, 500 * u.MeV],
+)
```

Comment:
```suggestion
TrackSelectorRanges = namedtuple(
    "TrackSelectorRanges",
    ["absEta", "pt"],
    defaults=[(None, None)] * 2,
)
```
except list all the parameters in `Examples/Algorithms/TruthTracking/ActsExamples/TruthTracking/TrackSelector.hpp`. Except `removeCharged` and `removeNeutral`, they can all be specified as ranges, hence my suggested name change.
The defaults are given as `None` so we can keep the defaults from the `TrackSelector` C++. We handle the `None`s in the argument passing below...

Reply:
one problem with this is that we except more input than we handle in the algorithm with might be confusing to the user

Reply:
> one problem with this is that we except more input than we handle in the algorithm with might be confusing to the user

Sorry, I don't understand this comment. We should only define parameters in `TrackSelectorRanges` that are defined in `TrackSelector`. Any of them *could* be used to select tracks as input for the vertexing.